content/channels/index.textile

```diff
@@ -767,6 +798,14 @@ channel.subscribe('myEvent') do |message|
 end
 ```
 
+```[realtime_python]  
+  realtime = AblyRealtime('{{API_KEY}}')
+  channel = realtime.channels.get('{{RANDOM_CHANNEL_NAME}}')
+  def listener(message):
+    print(f'Message received for {message.name}: ' + message.data)
```

Comment:
string concatenation and interpolation are both fine, but using them both on the same string is a little jarring 😄 
```suggestion
    print(f'Message received for {message.name}: {message.data}')
```

Reply:
Hah, yeah not sure what I was doing there! Fixed in [6d867e2](https://github.com/ably/docs/pull/1944/commits/6d867e2337199854e6dd60b46e944b748d96f52c)